react/Viewer/Readme.md

```diff
@@ -86,3 +71,43 @@ const onFileChange = (file, nextIndex) => setState({ currentFileIndex: nextIndex
   </DemoProvider>
 </div>
 ```
+
+### Using a worker for pdfjs
+
+For performance reasons, it is important to use a web worker when showing PDF files in the viewer. If you use webpack, you should add the following alias in your configuration :
+
+```patch
++ resolve: {
++   alias: {
++     'react-pdf$' : 'react-pdf/dist/entry.webpack.js'
```

Comment:
To be sure : the dollar is there to prevent an infinite loop in the alias resolution ?

Reply:
Nope! Luckily there's no infinite loop, even without dollar sign. But it makes sure that only imports of `react-pdf` are redirected to `react-pdf/dist/entry.webpack.js` — so if you `import from 'react-pdf/dist/something.js'`, you get `something.js` and not `entry.webpack.js`.